### ISSUE TITLE
Wrong indentation warning for one parameter lambdas without braces

### ISSUE THREAD

**Issue by jGleitz** [checkstyle/checkstyle#3273]
this code:

```java
return Files.find(folder, Integer.MAX_VALUE, (path, fileOpts) -> "pom.xml".equals(path.getFileName()))
	.map(pomXml -> folder.getParent().relativize(pomXml));
```

Produces the warning

> lambda bei Einrücktiefe 16 nicht an korrekter Einrücktiefe 17

(Rough translation: “Expected lambda at indentation level 16 to be at indentation level 17”)

However, this code does not:

```java
return Files.find(folder, Integer.MAX_VALUE, (path, fileOpts) -> "pom.xml".equals(path.getFileName()))
	.map((pomXml) -> folder.getParent().relativize(pomXml));
```

The code is correctly indented for my configuration, so I’d expect not to see a warning in both cases.

I could observe this behaviour at other places, too. Adding braces around the lambda parameter always fixed the warning.

My indentation configuration:
```xml
<module name="Indentation">
  <property name="severity" value="warning"/>
  <property name="lineWrappingIndentation" value="1"/>
</module>
```

**Comment by rnveach** [comment]
@jGleitz 
I am unable to reproduce your issue in the command line.
Please report the issue as described here http://checkstyle.sourceforge.net/report_issue.html#How_to_report_a_bug

**Comment by jGleitz** [comment]
> I am unable to reproduce your issue in the command line.

I’m sorry. I left out a `try … catch` that is necessary to reproduce the issue.

This file:

`Test.java`
```java
public class StrangeWarning {

	private Stream<Path> strangeBehaviour() {
		try {
			return Files.find(folder, Integer.MAX_VALUE, (path, fileOpts) -> "pom.xml".equals(path.getFileName()))
				.map(pomXml -> folder.getParent().relativize(pomXml));
		} catch (final IOException ioException) {
		}
	}
}
```

with this configuration:

`Checkstyle.xml`
```xml
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE module PUBLIC "-//Puppy Crawl//DTD Check Configuration 1.3//EN" "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">

<module name="Checker">
  <property name="severity" value="warning"/>
  <module name="TreeWalker">
    <property name="severity" value="warning"/>
    <property name="tabWidth" value="4"/>
    <module name="Indentation">
      <property name="severity" value="warning"/>
      <property name="lineWrappingIndentation" value="1"/>
    </module>
  </module>
</module>
```

produces this error: 
```
$ java -jar checkstyle-6.19-all.jar -c Checkstyle.xml Test.java
Starting audit...
[WARN] path/Test.java:6: lambda bei Einrücktiefe 16 nicht an korrekter Einrücktiefe 17 [Indentation]
Audit done.
```

While this file doesn’t:

`Test1.java`
```java
public class StrangeWarning {

	private Stream<Path> strangeBehaviour() {
		try {
			return Files.find(folder, Integer.MAX_VALUE, (path, fileOpts) -> "pom.xml".equals(path.getFileName()))
				.map((pomXml) -> folder.getParent().relativize(pomXml));
		} catch (final IOException ioException) {
		}
	}
}
```
```
$ java -jar checkstyle-6.19-all.jar -c Checkstyle.xml Test1.java 
Starting audit...
Audit done.
```

This file also doesn’t:

`Test2.java`
```
public class StrangeWarning {

	private Stream<Path> strangeBehaviour() {
		return Files.find(folder, Integer.MAX_VALUE, (path, fileOpts) -> "pom.xml".equals(path.getFileName()))
			.map(pomXml -> folder.getParent().relativize(pomXml));
	}
}
```
```
$ java -jar checkstyle-6.19-all.jar -c Checkstyle.xml Test2.java 
Starting audit...
Audit done.
```

I’d expect `Test.java` to also not produce a warning.
